docs/guide/consultation.md

```
@@ -0,0 +1,32 @@
+# 付费咨询
+
+我们计划提供付费技术咨询服务和外包服务，初步规划中。
+
+## 知识星球
+
+<img style="max-width: 300px; height: auto;" :src="zsxq" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)